addons/website/static/src/js/menu/edit.js

```diff
@@ -357,11 +357,21 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         // grep: COMPANY_TEAM_CONTENTEDITABLE
         const $extraEditableZones = $editableSavableZones.find('.s_company_team .o_not_editable img');
 
-        return $editableSavableZones.add($extraEditableZones).toArray();
+        return $editableSavableZones.add($extraEditableZones).toArray().concat(
+            // To make sure the selection remains bounded to the active tab,
+            // each tab is made non editable while keeping its nested
+            // oe_structure editable. This avoids having a selection range span
+            // over all further inactive tabs when using Chrome.
+            ...document.querySelectorAll('#wrapwrap .s_tabs > div > .s_tabs_main > .s_tabs_content > .tab-pane > .oe_structure')
```

Comment:
Mmh, there may be less fragile implementation than this... but not sure. In the end, I think that this structure ever change, the normal thing to do would be to grep s_tabs anyway so we would find this... :grimacing: Ideally a test should ensure this stays fixed but given the steps to reproduce... :grimacing: 

Thanks for finding a solution :+1: